### PR TITLE
Persist full Twitter attribution

### PR DESCRIPTION
**Pull request by leogrim**
@andrewconner

### CODE REVIEW

**Comment by andrewconner** [comment]
Works for me. We want everything? May be a bit large, but then again, not sure what we know we can throw out.

**Comment by leogrim** [comment]
I keep everything for Slack, I guess we may be happy to have it later on and a Tweet can never be that large :)

File: kifi-backend/modules/shoebox/app/com/keepit/model/RawKeep.scala [hunk 1]
```diff
@@ -29,10 +30,10 @@ case class RawKeep(
 }
 
 object RawKeep extends Logging {
-  def extractKeepSourceAttribtuion(keep: RawKeep): Option[SourceAttribution] = {
+  def extractKeepSourceAttribution(keep: RawKeep): Option[SourceAttribution] = {
     keep.source match {
       case KeepSource.twitterFileImport | KeepSource.twitterSync =>
-        val attrOpt = keep.originalJson.flatMap(js => PartialTwitterAttribution.fromRawTweetJson(js).asOpt)
+        val attrOpt = keep.originalJson.flatMap(_.asOpt[RawTweet].map(TwitterAttribution(_)))
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/app/com/keepit/model/RawKeep.scala:36]: :+1:

**Comment by andrewconner** [comment]
lgtm

**Pull request merged**